utils/config-dqTest-docQEbRNN.py

```diff
@@ -172,7 +172,7 @@ def load_parameters():
 
     # Model parameters
     # Perictor+Estimator
-    MODEL_TYPE = 'EncDoc'                 # Model to train. See model_zoo() for the supported architectures
+    MODEL_TYPE = 'EncSent'                 # Model to train. See model_zoo() for the supported architectures
```

Comment:
not obvious why this is `EncSent` when the MODEL_TYPE for config-dqTest-docQUbRNNEval.py is `EncDoc`